climada/engine/forecast.py

```diff
@@ -0,0 +1,897 @@
+                      'model_text': "CLIMADA IMPACT"}
+        f, _ = self._plot_imp_map(
+                             run_datetime,
+                             title=title_dict,
+                             cbar_label=('forecasted damage per gridcell [' +
+                                         self._impact[haz_ind].unit +
+                                         ']'),
+                             polygon_file=shapes_file
+                             )
+        if save_fig:
+            f.savefig(map_file_name_full)
+        if close_fig:
+            f.clf()
+            plt.close(f)
+
+    def _plot_imp_map(self, run_datetime, title, cbar_label, polygon_file=None):
+        # select hazard with run_datetime
+        if run_datetime is None:
+            run_datetime = self.run_datetime[0]
+        haz_ind = np.argwhere(np.isin(self.run_datetime, run_datetime))[0][0]
+        # tryout new plot with right projection
+        extend = 'neither'
+        crs_epsg = ccrs.Mercator()
+        value = self._impact[haz_ind].eai_exp
+        #    value[np.invert(mask)] = np.nan
+        coord = self._impact[haz_ind].coord_exp
+
+        # Generate array of values used in each subplot
+        array_sub = value
+        shapes = True
+        if not polygon_file:
+            shapes = False
+        proj = crs_epsg
+        var_name = cbar_label
+        geo_coord = coord
+        num_im, list_arr = u_plot._get_collection_arrays(array_sub)
+        list_tit = to_list(num_im, title, 'title')
+        list_name = to_list(num_im, var_name, 'var_name')
+        list_coord = to_list(num_im, geo_coord, 'geo_coord')
+
+        kwargs = dict()
+        kwargs['cmap'] = CMAP_IMPACT
+        kwargs['s'] = 5
+        kwargs['marker'] = ','
+        kwargs['norm'] = BoundaryNorm(
+            np.append(
+                np.append([0], [10 ** x for x in np.arange(0, 2.9, 2.9 / 9)]),
+                [10 ** x for x in np.arange(3, 7, 4 / 90)]),
+            CMAP_IMPACT.N,
+            clip=True
+            )
+
+        # Generate each subplot
+        fig, axis_sub = u_plot.make_map(num_im, proj=proj)
+        if not isinstance(axis_sub, np.ndarray):
+            axis_sub = np.array([[axis_sub]])
+        fig.set_size_inches(9, 8)
+        for array_im, axis, tit, name, coord in zip(list_arr,
+                                                    axis_sub.flatten(),
+                                                    list_tit,
+                                                    list_name,
+                                                    list_coord):
+            if coord.shape[0] != array_im.size:
+                raise ValueError("Size mismatch in input array: %s != %s." % \
+                                 (coord.shape[0], array_im.size))
+            # Binned image with coastlines
+            extent = u_plot._get_borders(coord)
+            axis.set_extent((extent), ccrs.PlateCarree())
```

Comment:
Why?? Above you defined `Mercator`. Please clean up and use the CLIMADA defaults consistently.